cursorless-talon/src/spoken_scope_forms.py

```diff
@@ -0,0 +1,76 @@
+from talon import Context
+
+from .csv_overrides import csv_get_ctx, csv_get_normalized_ctx
+
+
+def init_scope_spoken_forms(graphemes_talon_list: dict[str, str]):
+    create_flattened_talon_list(csv_get_ctx(), graphemes_talon_list, True)
+    create_flattened_talon_list(csv_get_normalized_ctx(), graphemes_talon_list, False)
```

Comment:
The normalized context is only used for the Cursorless Talon grammar tests so we can run test without user customization.